### PR TITLE
Prom scaler auth

### PR DESCRIPTION
**Pull request by marpio**
Update for https://github.com/kedacore/keda/issues/1559

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

**Comment by tomkerkhove** [comment]
LGTM, but waiting until the feature is merged

**Comment by zroubalik** [comment]
@marpio could you please rebuild this against 2.3 content? Thanks!

**Review by zroubalik — APPROVED** [review]

**Pull request merged**
